instrumentation/zipkin-opentracing.adoc

```diff
@@ -0,0 +1,31 @@
+:imagesdir: ../images
+
+:toc: macro
+:toc-title:
+
+== Opentracing
+
+TODO: Information about opentracing and how to use the experimental Java provider
+
+== Zipkin
+
+Zipkin project contains many instrumentation libraries for several programming languages, therefore it makes it easy to get tracing data for polyglot applications. It uses different approach as non-intrusive Java agent instrumentation. Developers have to use instrumented libraries or send tracing data directly to the server.  
+
+Hawkular APM is compatible with Zipkin collectors so any application instrumented with Zipkin library can report data to Hawkular APM. 
+
+==== Configuration
+Http collector is enabled by default. Kafka collector can be enabled by setting environmental variable `KAFKA_ZOOKEEPER`. It is zookeeper connect string, ex. `127.0.0.1:2181`.
```

Comment:
Yes I tnink so - otherwise new users may not understand what is meant by Http collector.